### DISCUSSION TITLE
(Android and Ios)The ability to easily change mania note size for people with ridiculously big finger

### DISCUSSION THREAD

**Discussion by IceLimeade** [ppy/osu#24684] [Ideas] [answered]
Tilte

**Comment by nagi-desuuu** [comment, marked as answer]
[This has been asked](https://github.com/ppy/osu/discussions/21388), and maybe you might want to look into [this](https://github.com/search?q=repo%3Appy%2Fosu+portrait&type=discussions)

**Reply by IceLimeade** [reply]
Oh
